add-on/panels/js/panel.jsx

```diff
@@ -340,7 +340,7 @@ loop.panel = (function(_, mozL10n) {
       if (this.props.userProfile && this.props.userProfile.email) {
         return (
           <div className="user-identity">
-            {loop.shared.utils.truncate(this.props.userProfile.email, 24)}
+            {this.props.userProfile.email}
```

Comment:
Got rid of the outer user-details containing DIV and the overflow works correctly because there is a single element to overflow now.